external/netty/playlist.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TKG/playlist.xsd">
+	<!--Add entries corresponding to your test similar to the example below so that
+		TestKitGen can produce the equivalent make command lines to be executed in the build
+	-->
+	<test>
+		<testCaseName>netty-test</testCaseName>
+		<command>docker run --rm $(EXTRA_DOCKER_ARGS) adoptopenjdk-netty-test:latest ; \
```

Comment:
To pass in parameters from Jenkinsfilebase and pull information from common_functions.sh, we employ external.sh instead of calling docker commands directly.  
You can take the [derby command](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/external/derby/playlist.xml#L31-L34) as an example, and replace your dir name (netty) where derby is in the command, 
```
<command> $(TEST_ROOT)$(D)external$(D)external.sh --run --tag ${DOCKERIMAGE_TAG} --version ${JDK_VERSION} --impl ${JDK_IMPL} --dir netty --reportdst $(REPORTDIR) --docker_args "$(EXTRA_DOCKER_ARGS)" ; \
		$(TEST_STATUS); \
		$(TEST_ROOT)$(D)external$(D)external.sh --clean --tag ${DOCKERIMAGE_TAG} --version ${JDK_VERSION} --impl ${JDK_IMPL} --dir netty
		</command>
```